check-required.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Exit immediately, but don't expose any environment variable values
+set +x
+set -e
```

Comment:
Through the dashboard possibly. A more pressing concern would be, if you're running this locally and you aren't logged in to Terminus and you don't have any of the global variables set, should the exit code be `0` or `1`. I think it should be `1`? But with the Terminus check first, it would be `0`.